scripts/run-integration-tests.sh

```diff
@@ -39,7 +43,7 @@ npm install -g yo generator-fabric-*.tgz
 rm -f generator-fabric-*.tgz
 
 chaincode_tests() {
-    LANGUAGES="go java javascript kotlin typescript"
+    LANGUAGES="go javascript typescript"
```

Comment:
why you take java and kotlin out?

Reply:
Bintray is currently broken, which stops Gradle from working at all. I'll be submitting another PR to turn them back on once Bintray is back up and running.